polyjuice-tests/src/helper.rs

```diff
@@ -359,7 +359,12 @@ impl PolyjuiceArgsBuilder {
 }
 
 pub fn setup() -> (Store, DummyState, Generator, u32) {
-    let _ = env_logger::try_init();
+    // If you want to watch the [contract debug] logs in Polyjuice,
+    // please change the log level from `info` to `debug`.
+    // then run `cargo test -- [test_filter] --nocapture`, 
+    // or run `RUST_LOG=debug cargo test -- [test_filter] --nocapture` directly
```

Comment:
better with `RUST_LOG=gw=debug`, the `gw` prefix can filter out a lot unrelated log.